### PR TITLE
fix: reverse listing of saved badges

### LINKED ISSUE #569: Latest saved sadges items are shown at bottom
**Describe the feature you'd like**
Currently, when you try to add a new Badge it gets added in reverse order as can be seen in Saved Badge Fragment

**Additional context**
The latest badge should be on the top of the list shown in recycler view of SavedBadge

### PR DESCRIPTION
**Pull request by refactor-droidyy**
Fixes #569

### CODE REVIEW

**Comment by adityastic** [comment]
@refactor-droidyy this would be useful if the files were read according to the date of creation. But right now it is just reads the files in  ascending order.

**Comment by refactor-droidyy** [comment]
@adityastic so do we need to add date data in badge Config JSON.

**Comment by adityastic** [comment]
@refactor-droidyy Config is just meant to store the config data, so let's just leave it for the important badge information. Let's sort this file list with date created then reverse it.

File: app/src/main/java/org/fossasia/badgemagic/util/StorageUtils.kt [hunk 1]
```diff
@@ -51,6 +51,13 @@ class StorageUtils(val context: Context) {
 
         externalStorageDir?.let {
             val files = File(externalStorageDir).listFiles() ?: return list
+            files.sortWith(Comparator<File> { a, b ->
+                when {
+                    a.lastModified() - b.lastModified() > 0 -> 1
+                    a.lastModified() - b.lastModified() < 0 -> -1
+                    else -> 0
+                }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/badgemagic/util/StorageUtils.kt]: `return a.lastModified() - b.lastModified()`

**Comment by refactor-droidyy** [comment]
@iamareebjamal, please review.

**Pull request merged**